wpeverest/user-registration
# UR-850 Enhance - Form Builder Design

### All Submissions:






### Changes proposed in this Pull Request:
This PR implements a new design of the form builder as per the [Design in Figma](https://www.figma.com/proto/zMjE7QEeroALWWK1EPD2GB/Form-Builder?node-id=1666-22407).

Major changes besides design:
1. Enable Strong Password option changed from select to radio-group.
2. Switch from Checkbox and Yes/No Select option to Toggle:
    a) Enable Strong Password, Enable Recaptcha.
    b) Required, Hide the label, Enable min/max date, Allow special characters in the username, etc.

### How to test the changes in this Pull Request:

1. Switch to this branch and verify if there are no inconsistencies in the design.
2. Do check the backward compatibilities, dynamic value update changes for the options with toggle.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Form Builder Design
